lib/commands/command.js

```diff
@@ -35,7 +35,14 @@ Command.prototype.execute = function (packet, connection) {
   }
 
   // TODO: don't return anything from execute, it's ugly and error-prone. Listen for 'end' event in connection
-  this.next = this.next(packet, connection);
+  try {
```

Comment:
I do not really like try/catches either, but I do not know another safe way to make sure no throwed errors are breaking the application implementing this library.

Maybe accept that bugs like this WILL break the app and that's that?